src/react-serializer.js

```diff
@@ -22,7 +25,8 @@ function getReactComponentSerializer() {
     }
     throw error;
   }
-  return value => renderer.create(value).toJSON();
+  return value =>
+    prettyFormat(renderer.create(value), { plugins: serializers });
```

Comment:
shouldn't we also use `jest-snapshot`'s serializers for `diffOptions` below?

Reply:
@thymikee Ah yes, good spot!